tools/run_tests/artifacts/artifact_targets.py

```diff
@@ -286,7 +286,7 @@ def pre_build_jobspecs(self):
     def build_jobspec(self):
         return create_docker_jobspec(
             self.name,
-            'tools/dockerfile/grpc_artifact_centos6_{}'.format(self.arch),
+            'tools/dockerfile/test/php73_zts_stretch_{}'.format(self.arch),
```

Comment:
Need to update this base docker image because `grpc_artifact_centos6` is too old for this purpose. It only has PHP 5.3.3 on it. We need PHP 7+.

Reply:
Fair enough but note that other languages generally don't use their "test" images for building the artifacts.
Mostly this is because the "test" docker images are kept on comparatively newer distros, which can cause trouble with portability of the binary artifacts they produce (if you build a library on a new linux distro, there might be backward compatibility issues when trying to run on older distros. That's why the umbrella "artifact" image is on such an old distro as centos6.
But if for PHP specifically building on stretch is fine (distribtests would ideally tell you that), than I'm fine with that.

Reply:
This is in some way necessary for PHP - the centos6 image only contains PHP 5.3.3 which is _way_ too old.

We are currently working separately on deprecating PHP 5 - it will take some work to make sure we get rid of PHP 5 across these testing frameworks.

So in the meantime, can we switch to this docker image for now? In the future (when we are working on deprecating PHP 5), I will look into switching to using `tools/dockerfile/grpc_artifact_linux_x64/Dockerfile`, but, changing that docker image from PHP 5 to PHP 7.